src/components/Drawer.js

```diff
@@ -229,7 +231,8 @@ class Drawer extends React.Component {
       },
       content: {
         backgroundColor: StyleConstants.Colors.WHITE,
-        height: '100%'
+        overflow: 'auto',
+        height: `calc(100% - ${HEADER_HEIGHT})`
```

Comment:
I texted this on safari mobile, so I know it works there. According to https://caniuse.com/#search=calc It seems that this should work on everything mobile outside of opera, which I am okay with 😬 .